srcpkgs/python3-dotty-dict/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-dotty-dict'
+pkgname=python3-dotty-dict
+version=1.3.0
+revision=1
+wrksrc="dotty_dict-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
+short_desc="Dictionary wrapper for quick access to deeply nested keys"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="MIT"
+homepage="https://github.com/pawelzny/dotty_dict"
+distfiles="${PYPI_SITE}/d/dotty_dict/dotty_dict-${version}.tar.gz"
+checksum=eb0035a3629ecd84397a68f1f42f1e94abd1c34577a19cd3eacad331ee7cbaf0
+
+# The tests aren't included in the tarball from PyPi
+do_check() {
+	:
+}
```

Comment:
We should either switch to the github tarball which contains tests or disable tests with `make_check=no` instead of overriding do_check. I would prefer switching to a github tarball.

Reply:
switched to the github tarball :+1: